publish-docs.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
```

Comment:
Ideally would have a license header.

Reply:
Good call. I too often start writing the code without thinking about the license.